### PR TITLE
Merge 1.7.6.x into 1.7.7.x - 07/07/2020

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Merge 1.7.6.x into 1.7.7.x
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | ~
| How to test?  | ~

Here is the list of files that had conflicts and need special attention:
- admin-dev/themes/default/template/helpers/list/list_action_delete.tpl (@PierreRambaud)
- classes/Currency.php (please check @atomiix)
- composer.lock (please check @PierreRambaud)
- classic/templates/catalog/product.tpl (please check @NeOMakinG)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20089)

### CODE REVIEW

**Comment by Progi1984** [comment]
@matks Please fix PHPCS :)

**Comment by matks** [comment]
> @matks Please fix PHPCS :)

Story of my life 😛

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -18,12 +18,50 @@ needs please refer to http://www.prestashop.com for more information.
 
 @author    PrestaShop SA <contact@prestashop.com>
 @copyright 2007-2020 PrestaShop SA and Contributors
+<<<<<<< HEAD
```
Comment (Progi1984) [inline docs/CHANGELOG.txt]: @matks :warning:

**Review by NeOMakinG — DISMISSED** [review]
Looks fine for product.tpl

**Review by Progi1984 — COMMENTED** [review]
A small fix to do and it's OK

**Review by PierreRambaud — DISMISSED** [review]

File: docs/CHANGELOG.txt (hunk 1 above)
Comment (matks) [reply docs/CHANGELOG.txt]: AAAaaaaAAAaaaah 😱 my bad !

Comment (matks) [reply docs/CHANGELOG.txt]: Fixed in https://github.com/PrestaShop/PrestaShop/pull/20089/commits/6153dfdd8318ae4c976ba3ff2f51ec22ec4d27a0

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]
LGTM!

**Comment by matks** [comment]
Thanks for review everyone 🎁

**Pull request merged**